.circleci/config.yml

```diff
@@ -84,23 +93,31 @@ workflows:
           name: helm_lint
       - hmpps/build_docker:
           name: build_docker
-          filters:
-            branches:
-              only:
-                - main
+      - request-dev-preview-approval:
```

Comment:
Just tidying but would remove the word `preview` here to match with later naming conventions as well as using underscores rather than dashes.